metadata-ingestion/src/datahub/ingestion/source/tableau.py

```diff
@@ -973,11 +1178,44 @@ def get_schema_metadata_for_custom_sql(
         )
         return schema_metadata
 
-    def _get_project(self, node):
-        if node.get("__typename") == "EmbeddedDatasource" and node.get("workbook"):
-            return node["workbook"].get("projectName")
-        elif node.get("__typename") == "PublishedDatasource":
-            return node.get("projectName")
+    def _get_project(self, ds):
```

Comment:
should this be called "get_project_path"? is that what this is returning?

Reply:
its extremely difficult to understand what this method _should_ be doing or returning.

especially becuse there is ANOTHER method below called "get_project_browse_path" and "get_project_path"?